_layouts/rnative-lesson.html

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+fw: true
+---
+<!-- ^idk what this does but im just putting it -->
+
+<script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+
+<div class="rnative-content-container">
+    <div>
+        <div class="rnative-content-card">
+            <h1 class="title text-2x no-margin">{{page.title}}</h1>
```

Comment:
Something to think about - there are two `h1`'s, one from the Github notes and one here. We should figure out a way to have only one!